lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1817,6 +1817,17 @@ def test_numerical_tolerance(self):
         self.assertAlmostEqual(result.coord("longitude").points[0], 28.5)
         self.assertAlmostEqual(result.coord("longitude").points[-1], 67.5)
 
+    def test_numerical_tolerance_wrapped(self):
+        # test the tolerance on the coordinate value causes modulus wrapping
+        # where appropriate
+        cube = create_cube(0.5, 3600.5, bounds=True)
+        lons = cube.coord("longitude")
+        lons.points = lons.points / 10
+        lons.bounds = lons.bounds / 10
+        result = cube.intersection(longitude=(-60, 60))
+        self.assertAlmostEqual(result.coord("longitude").points[0], -60.05)
+        self.assertAlmostEqual(result.coord("longitude").points[-1], 60.05)
```

Comment:
With iris 3.0.2, this would return a lon coord that looks like:
```
>>> result.coord('longitude')
DimCoord(array([-59.95, -59.85, -59.75, ..., 299.75, 299.85, 299.95]), bounds=array([[-60. , -59.9],
       [-59.9, -59.8],
       [-59.8, -59.7],
       ...,
       [299.7, 299.8],
       [299.8, 299.9],
       [299.9, 300. ]]), standard_name='longitude', units=Unit('degrees'))
```